crates/core/src/db/relational_db.rs

```diff
@@ -268,6 +268,12 @@ impl RelationalDB {
         self.inner.rollback_mut_tx_with_context(ctx, tx)
     }
 
+    #[tracing::instrument(skip_all)]
+    pub fn rollback_tx_with_context(&self, ctx: &ExecutionContext, tx: MutTxId) {
```

Comment:
Same naming question as in the other PR, maybe just `rollback_tx` for the non-test case.